.github/workflows/release.yaml

```diff
@@ -63,7 +63,6 @@ jobs:
         shell: bash
         run: |
           VERSION=${{ github.ref_name }}
```

Comment:
Need to write VERSION to environment variable?

Reply:
https://github.com/GreptimeTeam/greptimedb-operator/blob/develop/.github/workflows/release.yaml#L78

Reply:
Let's keep it to make it easy to configure in the future.